controllers/operator/apigateway_controller_test.go

```diff
@@ -254,20 +268,21 @@ var _ = Describe("API-Gateway Controller", func() {
 				Finalizers:        []string{ApiGatewayFinalizer},
 			},
 			}
-			apiRule := &gatewayv1beta1.APIRule{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "api-rule",
-					Namespace: "default",
-				},
-			}
-			oryRule := &oryv1alpha1.Rule{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "ory-rule",
-					Namespace: "default",
-				},
+
+			var rules []client.Object
+			// we initialize more than 5 objects, so we validate if we show only 5 in a condition
```

Comment:
I think we are missing the validation, that we show only 5. As far as I can see we only validate that the message contains 5.

Reply:
I don't quite get it. If the message contains less than 5, the loop will iterate over all objects. Since the condition message is set based on the size of the slice returned in API, checking number of elements as substring seems completely fine for me. Especially since the idea was to show only up to 5 values if the response has more than 5 elements.